src/Cake.XamarinStudio/Properties/Manifest.addin.xml

```diff
@@ -5,22 +5,32 @@
 			<File pattern="*.cake" />
 		</MimeType>
 	</Extension>
-	<Extension path="/MonoDevelop/SourceEditor2/SyntaxModes">
+    
+    <!-- Old way to declare a hightlight syntax
+    <Extension path="/MonoDevelop/SourceEditor2/SyntaxModes">
 		<Templates resource="Cake.XamarinStudio.Syntax.CakeSyntaxMode.xml" />
-	</Extension>
+	</Extension>-->
+    
+    <Extension path="/MonoDevelop/Ide/Editor/Bundles">       
```

Comment:
This extension point doesn't exist in Xamarin Studio so results in a warning. This will break compat with MD 6.x as well. I'll have to consider it's worth the effort still supporting 6.x now that 7.x is out..